### DISCUSSION TITLE
直接下载网易云音乐中歌曲MP3格式的方法 | Mengke's blog - Mengke's coding journey

### DISCUSSION THREAD

**Discussion by giscus** [mk965/mengke.me#6] [Announcements]
# 直接下载网易云音乐中歌曲MP3格式的方法 | Mengke's blog - Mengke's coding journey

直接下载网易云音乐中歌曲MP3

https://www.mengke.me/blog/202307/neteasy_cloud_music_download_mp3

**Comment by 00kka** [comment]
感谢提供的方法👍

**Comment by pe1q1** [comment]
肥肠感谢，这个方法好使666
